### ISSUE TITLE
Evaluate the implementation

### ISSUE THREAD

**Issue by Hannarasmussen** [Hannarasmussen/INF264-Project-1-H25#9]
Evaluate the performance of the selected decision tree and random forest models on the test set. Report
the performance of the models using an appropriate performance measure and compare the results
of the two models. You should also compare your implementation to an existing decision tree and
random forest implementation, such as the DecisionTreeClassifier and RandomForestClassifier
classes from sklearn.
Remember to also perform model selection for the existing implementations to ensure a fair comparison.
Report the performance of the existing implementations on the test set and compare the results to
your own implementations.
